src/Icicle/Sea/FromAvalanche/Prim.hs

```diff
@@ -142,6 +142,8 @@ seaOfPrimUpdate p
  = case p of
      PrimUpdateArrayPut t
       -> prefixOfValType (ArrayT t) <> "put"
+     PrimUpdateArrayPut2 t1 t2
+      -> prefixOfValType (PairT (ArrayT t1) (ArrayT t2)) <> "put2"
```

Comment:
actually, no. This should become a C statement expression with "put", I was just chucking this here for now.